recipes/temporary/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "temporary" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "6a56e5cc0b5df7c1f55cefefba9fb5f71cc13332b2929f2c8976db9be3bed773" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - contextlib2
+    - pathlib2
+  run:
+    - python
+    - contextlib2
+    - pathlib2
+
+test:
+  imports:
+    - temporary
+
+about:
+  home: https://github.com/themattrix/python-temporary
+  license: MIT
+  license_family: MIT
```

Comment:
Hmm, upstream isn't using MANIFEST.in, but a license is included in the code, so I have added `license_file: LICENSE`. Hope that's ok.